core/jazz_es-kinesis-log-streamer/components/utils.js

```diff
@@ -20,13 +20,80 @@
   @version: 1.0
 **/
 
+const AWS = require('aws-sdk');
+const Uuid = require("uuid/v4");
+const request = require('request');
 const crypto = require('crypto');
 const logger = require("../components/logger.js");
-const config = require("../config/global_config.json")
+const config = require("../config/global_config.json");
 var truncate = require('unicode-byte-truncate');
 
 // Helper functions
 
+function assumeRole(configData, serviceData){
+  var isPrimary, roleArn;
+  if(serviceData){
+  	isPrimary = checkIsPrimary(serviceData.deployment_accounts[0].accountId, configData);
+  	roleArn = getRolePlatformService(serviceData.deployment_accounts[0].accountId, configData);
+  } else {
+    // this is to handle older services which had '-' in their naming convention, thus serviceData will be undefined for them
```

Comment:
This scenario should not happen, but gracefully return from here after logging error and return error.

Reply:
remove this hack for now.

Reply:
Done.